wrlx-image/content.md

```diff
@@ -0,0 +1,45 @@
+# What is wrlx-image?
+
+It is short for Wind River Linux image, which is a small docker image built from [Wind River Linux LTS18](https://github.com/WindRiver-Labs/wrlinux-x), it is based on glibc and busybox, can be expanded by package manager opkg. It supports a lot of most commonly used packages, such as mariadb, apache2, gcc and so on. And busybox can be replaced by related tools easily.
+
+%%LOGO%%
+
+# How to use this image
+
+## Usage
+
+Use like you would any other base image:
+
+```dockerfile
+FROM %%IMAGE%%:10.18.0011
+RUN opkg update
+RUN opkg install python3-django
```

Comment:
If `opkg update` is similar to `apt-get update` (and only updates the list of available packages), then these should be combined:

```suggestion
RUN opkg update && opkg install python3-django
```

---

If `opkg update` is more similar to `apt-get upgrade` (updating packages themselves, not just the list of available packages), then it should be dropped instead -- we encourage base image maintainers to update their base image periodically to get updated packages downstream to users:

```suggestion
RUN opkg install python3-django
```